doc/changes/changes_2.0.0.md

```diff
@@ -1,7 +1,6 @@
-# dynamodb-virtual-schema 2.0.0, released 2020-09-XX
+# dynamodb-virtual-schema 2.0.0, released 2020-10-05
```

Comment:
Today is 9th October :) if you plan to release it today

Reply:
```suggestion
# dynamodb-virtual-schema 2.0.0, released 2020-XX-XX
```